src/coins.h

```diff
@@ -246,6 +249,9 @@ class CCoinsViewCache : public CCoinsViewBacked
     /* Cached dynamic memory usage for the inner Coin objects. */
     mutable size_t cachedCoinsUsage;
 
+    // Necessary so that we can write directly into cacheCoins during snapshot load.
+    friend ChainstateManager;
```

Comment:
in 3a547327446a3608ef2af25a8d25a42954fc5116

With the introduction of `EmplaceCoinInternalDANGER()` you can remove this line now, right?

Reply:
Ugh, embarrassing. Thanks.